api/types.proto

```diff
@@ -922,5 +938,8 @@ message RootRotation {
 	bytes ca_cert = 1 [(gogoproto.customname) = "CACert"];
 	bytes ca_key = 2 [(gogoproto.customname) = "CAKey"];
 	// cross-signed CA cert is the CACert that has been cross-signed by the previous root
-	bytes cross_signed_ca_cert = 3 [(gogoproto.customname) = "CrossSignedCACert"];;
+	bytes cross_signed_ca_cert = 3 [(gogoproto.customname) = "CrossSignedCACert"];
+	// ForceRotationVersion matches the Cluster Spec's CAConfig's ForceRotation counter.
+	// It indicates when the current to-be-rotated CA cert and key were generated (or updated).
+	uint64 force_rotation_version = 6;
```

Comment:
Never mind, read the comments... But I'm still surprised we would need to keep track of both. Will read the code next.